crates/ironrdp-connector/Cargo.toml

```
@@ -30,9 +30,9 @@ rand_core = { version = "0.6", features = [
 sspi.workspace = true
 tracing.workspace = true
 url = "2.5"
-picky-asn1-der = "0.5.0"
-picky-asn1-x509 = "0.13.0"
-picky = "7.0.0-rc.10"
+picky-asn1-der = "0.5.1"
+picky-asn1-x509 = "0.14.1"
```

suggestion: To avoid dependency over-constraining:
```suggestion
picky-asn1-der = "0.5"
picky-asn1-x509 = "0.14"
```

---

It's not a so strict constaint, cargo will update to >=0.5.1 <0.6, but ok

---

"image" is likely to be already downloaded by a rust dev & available in various distros. otoh, bmp is an outdated format.

It's true, that image is a bigger crate (cargo bloat):

 0.1%   0.3%   26.5KiB image
 0.0%   0.0%    3.4KiB bmp

But it's so marginal compared to:

 7.3%  16.8%    1.5MiB sspi
 4.8%  10.9% 1013.8KiB aws_lc_sys
 4.0%   9.2%  857.7KiB std
 3.1%   7.1%  663.3KiB reqwest
 2.1%   4.8%  441.8KiB rustls
 1.8%   4.2%  390.2KiB regex_syntax
 1.5%   3.5%  325.7KiB h2
 1.4%   3.3%  304.6KiB ironrdp_pdu
 1.4%   3.1%  290.3KiB ironrdp_connector
 1.1%   2.5%  235.2KiB hickory_proto
 1.1%   2.5%  232.9KiB picky
...

---

> It's not a so strict constaint, cargo will update to >=0.5.1 <0.6, but ok

Sure, but let’s avoid accumulate this kind of unnecessary constraints. I already got compilation errors due to crates specifying the patch version for every single dependency.